README.md

```diff
@@ -36,6 +35,8 @@ If you are not familiar with the purpose and basic use of Terraform yet, we reco
 
 - **Azure subscription**: If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio) before you begin.
 
+  Required permissions: you will create a service principal that is `Contributor` rights on the subscription
```

Comment:
Here I wanted to add a few more points on the types of resources it is going to create
@sphuber could you provide a list?

Reply:
It will create:
* Solution
* Kubernetes cluster
* Log Analytics workspace

Reply:
![image](https://user-images.githubusercontent.com/6992332/193669843-65bfed15-8b43-4ce9-ab89-e91f4a6285d2.png)

Reply:
Thanks, let's add this in one of the next PRs